app/views/catalog/_show_actions_box_collection.html.erb

```diff
@@ -1 +1,40 @@
-<% # intentionally left blank to not render the actions box for collections %>
+<div class='al-show-actions-box col-md-3'>
```

Comment:
Is it possible that we could get away without this partial and just use the default one? Requestability logic should constrain the request button from being shown here I think.